spec/requests/api/orders_spec.rb

```diff
@@ -153,6 +149,30 @@
               expect(orders.first["id"]).to eq order_dist_2.id
             end
           end
+
+          context "and queried by cart state" do
+            let!(:order_empty) {
+              create(:order_with_line_items, line_items_count: 0)
+            }
+
+            let!(:order_not_empty) {
+              create(:order_with_line_items, line_items_count: 1)
+            }
+
+            let!(:order_not_empty_no_address) {
+              create(:order_with_line_items, line_items_count: 1, bill_address_id: nil, ship_address_id: nil)
+            }
+
+            let(:'q[state_eq]') { "cart" }
+
+            run_test! do |response|
+              expect(response).to have_http_status(200)
```

Comment:
This test is already in the `response(200, 'get orders')` block which will test this automatically. You don't need this line.

Reply:
Thanks - indeed @mkllnk . I've removed it, squased, rebased.